HunterHow - invalid memory address or nil pointer dereference

I'm getting the following when trying to run hunterhow:

```console
$/home/kali/go/bin/uncover -q hunterhow.query -e hunterhow -l 1000 -o hunterhow.txt

[INF] Current uncover version v1.0.3 (latest)
[hunterhow] couldn't create new session: no default rate limit found for engine hunterhow
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xc9cbd8]

goroutine 36 [running]:
github.com/projectdiscovery/uncover/uncover/agent/hunterhow.(*Agent).Query(0xc0004b5350?, 0xe3f9d9?, 0xdd7d93?)
        /home/kali/go/pkg/mod/github.com/projectdiscovery/uncover@v1.0.3/uncover/agent/hunterhow/hunterhow.go:24 +0x18
github.com/projectdiscovery/uncover/runner.(*Runner).Run.func1({0x1031a70, 0x162eaf0}, 0xc00040e450?)
        /home/kali/go/pkg/mod/github.com/projectdiscovery/uncover@v1.0.3/runner/runner.go:184 +0x4af
created by github.com/projectdiscovery/uncover/runner.(*Runner).Run
        /home/kali/go/pkg/mod/github.com/projectdiscovery/uncover@v1.0.3/runner/runner.go:165 +0x1cdb
```

Am I missing something? Do I need to edit something?
Also, there's no HunterHow's key section in the provider-config.yaml file.

--- a/sources/agent/hunterhow/hunterhow.go
+++ b/sources/agent/hunterhow/hunterhow.go
@@ -37,7 +37,7 @@ func (agent *Agent) Query(session *sources.Session, query *sources.Query) (chan 
 		for {
 			hunterhowRequest := &Request{
 				Query:    query.Query,
-				PageSize: query.Limit,
+				PageSize: Size, // max size is 100
 				Page:     pageQuery,
 			}
 
@@ -75,7 +75,7 @@ func (agent *Agent) query(URL string, session *sources.Session, results chan sou
 		results <- sources.Result{Source: agent.Name(), Error: err}
 		return nil
 	}
-	if apiResponse.Code != 200 {
+	if apiResponse.Code != http.StatusOK {
 		results <- sources.Result{Source: agent.Name(), Error: errors.New(apiResponse.Message)}
 		return nil
 	}

--- a/sources/session.go
+++ b/sources/session.go
@@ -26,7 +26,7 @@ var DefaultRateLimits = map[string]*ratelimit.Options{
 	"netlas":     {Key: "netlas", MaxCount: 1, Duration: time.Second},
 	"criminalip": {Key: "criminalip", MaxCount: 1, Duration: time.Second},
 	"publicwww":  {Key: "publicwww", MaxCount: 1, Duration: time.Minute},
-	"hunterhow":  {Key: "hunterhow", MaxCount: 1, Duration: time.Second},
+	"hunterhow":  {Key: "hunterhow", MaxCount: 1, Duration: 3 * time.Second},
 }
 
 // Session handles session agent sessions

